src/Raven.Server/Utils/ChangeVectorUtils.cs

```diff
@@ -144,8 +144,7 @@ public static (bool IsValid, string ChangeVector) TryUpdateChangeVector(string n
                 // compare the strings instead of parsing to int
                 if (existingEtag >= etag)
                 {
-                    //nothing to do
-                    return (false, null);
+                    ThrowConflictingEtag(etag, oldChangeVector, existingEtag, dbIdInBase64, nodeTag);
```

Comment:
This is _valid_. 
That isn't where we need to check this, only on the actual call site where we know this is the case.